### PR TITLE
calentre-client-29: create enter details section UI

### PR DESCRIPTION
**Pull request by luisflores93**

### CODE REVIEW

File: src/app/[user-name]/layout.tsx [hunk 1]
```diff
@@ -4,7 +4,10 @@ import type { Metadata } from 'next';
 
 interface Props {
   children: React.ReactNode;
-  params: { 'user-name'?: string };
+  params: {
+    'user-name'?: string;
+    'event-name'?: string;
```
Comment (fiizzy) [inline src/app/[user-name]/layout.tsx]: Is there a use case where the user-name and event-name would be null?

File: src/components/Icons/CalendarCheck.tsx
Comment (fiizzy) [inline src/components/Icons/CalendarCheck.tsx:1]: Do you manually have to set the `color` props within the SVG files? 🤔

Comment (fiizzy) [reply src/components/Icons/CalendarCheck.tsx:1]: Thinking about how sustainable that's going to be as we add more svgs

File: src/types/global.d.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 interface PageProps {
-  params?: {
-    'user-name': string;
-    'event-name': string;
+  params: {
+    'user-name'?: string;
```
Comment (fiizzy) [inline src/types/global.d.ts]: Same as before. Are we expecting to have null values?

**Review by fiizzy — COMMENTED** [review]
Great work refactoring this @luisflores93 ! 
Just a few clarifications. I mostly just want to hear your thoughts.

File: src/app/[user-name]/layout.tsx (hunk 1 above)
Comment (luisflores93) [reply src/app/[user-name]/layout.tsx]: well, now that I think, this does not make sense 🤔

Comment (fiizzy) [reply src/app/[user-name]/layout.tsx]: Great! Just ping me when it's updated and I'll review ASAP.

File: src/components/Icons/CalendarCheck.tsx
Comment (luisflores93) [reply src/components/Icons/CalendarCheck.tsx:1]: > Do you manually have to set the `color` props within the SVG files? 🤔

yeah I agree, I'll update it, we can use colors from tailwind theme, so we can avoid writing it manually

Comment (luisflores93) [reply src/components/Icons/CalendarCheck.tsx:1]: I'll update other icons on another PR

**Review by fiizzy — APPROVED** [review]
LGTM

**Pull request merged**